src/components/Header/Header.jsx

```diff
@@ -10,33 +13,55 @@ import {
   ButtonsBox,
   HeaderLogo,
 } from "./styled";
+import { remove } from "../../utils/redux/features/addUser/userSlice";
 
-const Header = () => (
-  <HeaderBox>
-    <AppBar position="static">
-      <HeaderToolbar>
-        <HeaderLink to={ROUTES.myNotes}>
-          <HeaderLogo variant="h5" component="span">
-            Notes app
-          </HeaderLogo>
-        </HeaderLink>
-        <ButtonsBox>
+const Header = ({ values }) => {
+  const dispatch = useDispatch();
+  const user = useSelector((state) => state.user);
+  const history = useHistory();
+  const redirectToSignIn = () => {
+    history.push("/sign-in");
```

Comment:
Can you please use path from contants? It will help us in the future with updating routes (one source of truth) and with possible typos(avoid them)